modules/ngx_http_cbor_ic_modules/crc32.h

```diff
@@ -0,0 +1,3 @@
+#include <stdio.h>
+
+unsigned int ic_crc32(const unsigned char *message, size_t len);
```

Comment:
I wanted to be canonical and use `.h` for signatures and `.c` for code :) But I don't care really, can do

Reply:
Also it's a bit flaky since if you include e.g. `crc32.h` containing function implementation from some other 2+ `.c` files you'll get a link time failure since both of them would export that symbol. We can define it as static but in this case we'll end up having two copies of function.

In this case it's used only in one place (well, in tests also), but if something changes -> linking will fail.